docs/examples/control_looping_UWY_UFS_CHEM.yaml

```diff
@@ -0,0 +1,127 @@
+
+# Example pairing script that pairs University of Wyoming sonde data with UFS-Chem 
+
+analysis:
+  start_time: '2023-06-27 23:00:00' #UTC
+  end_time: '2023-06-28-23:59:00' #UTC
+  output_dir:  ./output/uwy_ufschem
```

Comment:
Can you add this directory to the GitHub page like the other exampes and add a .gitignore like you did for ish and ishlite and airnow with UFS-Chem?